admin/upgrades/list-table.php

```diff
@@ -0,0 +1,125 @@
+<?php
+namespace CBOX\Admin\Upgrades;
+
+use CBOX\Upgrades\Upgrade_Registry;
+
+if ( ! class_exists( '\WP_List_Table' ) ) {
+	require_once ABSPATH . 'wp-admin/includes/class-wp-list-table.php';
+}
+
+class List_Table extends \WP_List_Table {
+
+	/**
+	 * Constructor.
+	 *
+	 * @return void
+	 */
+	public function __construct() {
+		parent::__construct( [
+			'plural'   => 'upgrades',
+			'singular' => 'upgrade',
+			'ajax'     => false
+		] );
+	}
+
+	/**
+	 * Message to be displayed when there are no items.
+	 *
+	 * @return void
+	 */
+	public function no_items() {
+		_e( 'No upgrades found.', 'commons-in-a-box' );
```

Comment:
Here and elsewhere, let's use the escaped wrappers `esc_html_e()` etc.

Reply:
Updated. I think I covered all the cases.